cloud/api-gateway/src/main/resources/application.yaml

```diff
@@ -1,3 +1,26 @@
 spring:
   application:
-    name: api-gateway
\ No newline at end of file
+    name: api-gateway
+  cloud:
+      gateway:
+        routes:
+          - id: user-manager
+            uri: http://localhost:8081
+            predicates:
+              - Path=/api/v1/customers/**,/api/v1/merchants/**
+          - id: payment-manager
+            uri: http://localhost:8082
+            predicates:
+              - Path=/api/v1/wallets/**,/api/v1/payments/**
+          - id: notification-manager
```

Comment:
for internal use. Remove it

Reply:
remove notfication service or paths in payment-manager?

Reply:
i thought these paths should be able for user to get wallet and provide payment and so on

Reply:
Users cannot directly send requests to notification manager.